osmaxx/conversion/converters/converter_gis/bootstrap/bootstrap.py

```diff
@@ -1,30 +1,28 @@
 import glob
-import subprocess
-
 import os
 
 from osmaxx.conversion.converters.converter_gis.helper.default_postgres import get_default_postgres_wrapper
-from osmaxx.utils import changed_dir
+from osmaxx.conversion.converters.converter_gis.helper.osm_importer import OSMImporter
+from osmaxx.utils import changed_dir, polyfile_helpers
 
 
-def boostrap(pbf_file_path):
+def boostrap(area_polyfile_string):
     with changed_dir(os.path.dirname(__file__)):
-        bootstrapper = BootStrapper(pbf_file_path=pbf_file_path)
+        bootstrapper = BootStrapper(area_polyfile_string)
         bootstrapper.bootstrap()
```

Comment:
Could the `with changed_dir(...)` call be moved into `BootStrapper.bootstrap(...)`? I don't think the `BootStrapper` initialization is sensitive to the current working dir.